packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.styles.ts

```diff
@@ -0,0 +1,184 @@
+    box-shadow: ${shadow4};
+    border: 2px;
+    pointer-events: none;
+  }
+  .tooltip-line {
+    padding-inline-start: 8px;
+    height: 50px;
+    border-inline-start: 4px solid;
+  }
+  .tooltip-legend-text {
+    font-size: 13px;
+    color: theme.semanticColors.bodyText;
+    text-align: start;
+  }
+  .tooltip-data-y {
+    font-weight: bold;
+    font-size: 30px;
+    text-align: start;
+    lineheight: 36px;
+    margin-top: 12px;
+  }
+  .bar {
+    opacity: 1;
+  }
+  .bar.inactive {
+    opacity: 0.1;
+  }
+  .bar:focus {
+    outline: none;
+    stroke-width: 2px;
+    stroke: black;
+  }
+  .svg-chart {
+    display: block;
+    overflow: visible;
+  }
+  .chart-title {
+    ${typographyBody1Styles}
+    display: flex;
+    justify-content: space-between;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+    display: block;
+    color: ${colorNeutralForeground1};
+    margin-bottom: ${spacingHorizontalSNudge};
+  }
+  .legendcontainer {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding-top: ${spacingVerticalL};
+    width: 100%;
+    align-items: center;
+    margin: -8px 0 0 -8px;
+  }
+  .legend {
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    border: none;
+    padding: ${spacingHorizontalS};
+    background: none;
+    text-transform: capitalize;
+  }
+  .legend-rect {
+    width: 12px;
+    height: 12px;
+    margin-inline-end: ${spacingHorizontalS};
+    border: 1px solid;
+  }
+  .legend-text {
+    ${typographyCaption1Styles}
+    color: ${colorNeutralForeground1};
+  }
+  .legend.inactive .legend-rect {
+    background-color: transparent !important;
+  }
+  .legend.inactive .legend-text {
+    opacity: 0.67;
+  }
+  .bar-label {
+    ${typographyBody1StrongStyles}
+    fill: ${colorNeutralForeground1};
+  }
+  .chart-title-div {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+  }
+  .ratio-numerator {
+    ${typographyBody1StrongStyles}
+    color: ${colorNeutralForeground1};
+  }
+  .ratio-denominator {
+    ${typographyBody1StrongStyles}
+    color: ${colorNeutralForeground1};
+    font-weight: bold;
+  }
+  .benchmark-container {
+    position: relative;
+    height: 7px;
+    margin-top: -3px;
+  }
+  .triangle {
+    width: 0;
+    height: 0;
+    border-left: 4px solid transparent;
+    border-right: 4px solid transparent;
+    border-bottom: 7px solid;
+    border-bottom-color: ${colorNeutralStrokeAccessible};
+    margin-bottom: 4px;
+    position: absolute;
+  }
+  .chart-data-text {
+    ${typographyBody1StrongStyles}
+    color: ${colorNeutralForeground1};
+  }
+`.withBehaviors(
+  forcedColorsStylesheetBehavior(css`
+    .legend-rect,
+    .tooltip-line,
+    .triangle {
+      forced-color-adjust: none;
+    }
+    .tooltip-legend-text,
+    .tooltip-content-y {
+      forced-color-adjust: auto;
+      color: rgb(255, 255, 255);
+    }
+    .bar-label {
+      fill: white !important;
+    }
```

Comment:
Again we should be using CSS System Colors, making sure this working in both light and dark. 

you're specifying colors in both RGB and a named color, please be consistent and/or use a CSS System Color or a token.

Reply:
@krkshitij could you address this comment.

Reply:
replaced RGB and named colors with CSS system colors for high contrast mode in #33622